code/testapp/src/main/java/com/adobe/marketing/mobile/core/testapp/services/NetworkServiceCard.kt

```diff
@@ -0,0 +1,52 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+ */
+package com.adobe.marketing.mobile.core.testapp.services
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.adobe.marketing.mobile.core.testapp.showAlert
+import com.adobe.marketing.mobile.services.HttpMethod
+import com.adobe.marketing.mobile.services.NetworkRequest
+import com.adobe.marketing.mobile.services.ServiceProvider
+
+@Composable
+fun NetworkServiceCard() {
+
+    Card(modifier = Modifier.padding(16.dp), elevation = 8.dp) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(text = "Network Services", modifier = Modifier.padding(16.dp))
+
+            Button(onClick = {
+                val request = NetworkRequest("https://www.adobe.com", HttpMethod.GET, null, null, 5000,5000)
+                ServiceProvider.getInstance().networkService?.connectAsync(request){ connection ->
+                    val status = if (connection != null) "valid connection" else "null connection"
+                    showAlert("Privacy Status: $status")
```

Comment:
Connection status.
nit: I also suggest also showing the http status code if the connection is not null.